diff --git a/script/bootstrap b/script/bootstrap
index 81fb0cf..c12bada 100755
--- a/script/bootstrap
+++ b/script/bootstrap
@@ -15,13 +15,9 @@ echo "Cleaning up Clojure archive..."
 rm clojure-1.3.0.zip
 
 echo "Fetching Google Closure library..."
-mkdir -p closure/library
-cd closure/library
-curl -O -s http://closure-library.googlecode.com/files/closure-library-20110323-r790.zip
-unzip -qu closure-library-20110323-r790.zip
-echo "Cleaning up Google Closure library archive..."
-rm closure-library-20110323-r790.zip
-cd ..
+mkdir -p closure
+cd closure
+svn checkout http://closure-library.googlecode.com/svn/trunk/ library/
 
 echo "Fetching Google Closure compiler..."
 mkdir -p compiler
